networks/mainnet.json

```
@@ -27,14 +28,29 @@
     "vETH": "0xf508fCD89b8bd15579dc79A6827cB4686A3592c8",
     "vLTC": "0x57A5297F2cB2c0AaC9D554660acd6D385Ab50c6B",
     "vXRP": "0xB248a295732e0225acd3337607cc01068e3b9c10",
+    "vTRX": "0xC5D3466aA484B040eE977073fcF337f2c00071c1",
     "Timelock": "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396",
     "GovernorAlpha": "0x406f48f47D25E9caa29f17e7Cfbd1dc6878F078f",
     "VenusPriceOracle": "0x516c18DC440f107f12619a6d2cc320622807d0eE",
-    "XVSVault": "0x0000000000000000000000000000000000000000",
-    "XVSVaultProxy": "0x0000000000000000000000000000000000000000",
-    "XVSStore": "0x0000000000000000000000000000000000000000",
     "WBNB": "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c",
-    "pancakeFactory": "0xcA143Ce32Fe78f1f7019d7d551a6402fC5350c73"
+    "pancakeFactory": "0xcA143Ce32Fe78f1f7019d7d551a6402fC5350c73",
+    "XVSVault": "0x6eF49b4e0772Fe78128F981d42D54172b55eCF9F",
+    "XVSVaultProxy": "0x051100480289e704d20e9DB4804837068f3f9204",
+    "XVSStore": "0x1e25CF968f12850003Db17E0Dba32108509C4359",
+    "SnapshotLens": "0x9802448Af040DA880Bf51bCbe4f2A4205ebC6d2C",
+    "VTreasury": "0x0000000000000000000000000000000000000000",
+    "GovernorAlpha2": "0x388313BfEFEE8ddfeAD55b585F62812293Cf3A60",
+    "GovernorBravoDelegate": "0x0000000000000000000000000000000000000000",
+    "GovernorBravoDelegator": "0x2d56dC077072B53571b8252008C60e945108c75a",
+    "Base200bps_Slope2000bps_Jump2000bps_Kink90": "0x0000000000000000000000000000000000000000",
+    "VRTVault": "0x0000000000000000000000000000000000000000",
+    "VRTVaultProxy": "0x98bF4786D72AAEF6c714425126Dd92f149e3F334",
+    "VRTConverter": "0xF0fCB229031A386C672e291cE183390c1a7705dA",
+    "VRTConverterProxy": "0x92572fB60f4874d37917C53599cAe5b085B9Facd",
+    "XVSVesting": "0x0000000000000000000000000000000000000000",
+    "XVSVestingProxy": "0xb28Dec7C7Ac80f4D0B6a1B711c39e444cDE8B2cE",
+    "ComptrollerLens": "0x0000000000000000000000000000000000000000",
+    "Liquidator": "0x0000000000000000000000000000000000000000"
```

(This one is an openzeppelin proxy)

```suggestion
    "Liquidator": "0x0870793286aaDA55D39CE7f82fb2766e8004cF43"
```